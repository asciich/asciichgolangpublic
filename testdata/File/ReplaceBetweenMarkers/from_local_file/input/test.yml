---

defined: before

# REPLACE_BETWEEN_MARKERS START source=./stages.txt
# REPLACE_BETWEEN_MARKERS END

defined_again: after