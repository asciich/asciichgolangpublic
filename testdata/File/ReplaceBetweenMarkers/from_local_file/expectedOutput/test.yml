---

defined: before

# REPLACE_BETWEEN_MARKERS START source=./stages.txt
stages:
- a
- b
- c

# REPLACE_BETWEEN_MARKERS END

defined_again: after